### PR TITLE
Update ci.yml

### PR DESCRIPTION
**Pull request by jason-fox**
Add Node 14 to unit-test matrix

### CODE REVIEW

File: doc/deprecated.md [hunk 1]
```diff
@@ -13,7 +13,8 @@ longer. In particular:
 A list of deprecated features and the version in which they were deprecated follows:
 
 -   Support to NGSI v1.
--   Support to Node.js v4 in iotagent-node-lib 2.8.1. The use of Node.js v10 is highly recommended.
+-   Support to Node.js v4 in iotagent-node-lib 2.8.1.
+-   Support to Node.js v10 in iotagent-ul 2.15.0. The use of Node.js v12 is highly recommended.
```
Comment (fgalan) [inline doc/deprecated.md:17]: For the sake of completeness, maybe information about versions between v4 and v10 should be included (it should be easy to know looking at https://github.com/telefonicaid/iotagent-node-lib/releases )... although maybe it doesn't worth the time.

Not sure how it's better...

Comment (jason-fox) [reply doc/deprecated.md:17]: Fixed 9b5f768

Comment (jason-fox) [reply doc/deprecated.md]: Fixed 479f120

File: doc/deprecated.md [hunk 2]
```diff
@@ -13,7 +13,12 @@ longer. In particular:
 A list of deprecated features and the version in which they were deprecated follows:
 
 -   Support to NGSI v1.
--   Support to Node.js v4 in iotagent-node-lib 2.8.1. The use of Node.js v10 is highly recommended.
+-   Support to Node.js v4 in iotagent-node-lib 2.8.1.
+-   Support to Node.js v6 in iotagent-node-lib 2.9.0.
+-   Support to Node.js v8 in iotagent-node-lib 2.12.0.
```
Comment (fgalan) [inline doc/deprecated.md:18]:
```suggestion
-   Support to Node.js v4 in iotagent-node-lib 2.8.1 (finally removed in 2.9.0)
-   Support to Node.js v6 in iotagent-node-lib 2.9.0 (finally removed in 2.10.0)
-   Support to Node.js v8 in iotagent-node-lib 2.12.0 (finally removed in 2.13.0)
```

File: doc/deprecated.md [hunk 3]
```diff
@@ -31,8 +36,10 @@ information in the case you want to use old versions:
...
-| --------------------- | ----------------------------------------------------- | ----------------------------- |
-| NGSI v1 API           | Not yet defined                                       | Not yet defined               |
-| Support to Node.js v6 | Not yet defined but it will be done by May 2019       | Not yet defined               |
-| Support to Node.js v4 | 2.8.1                                                 | December 19th, 2018           |
+| **Removed feature**    | **Last iotagent-node-lib version supporting feature** | **That version release date** |
+| ---------------------- | ----------------------------------------------------- | ----------------------------- |
+| NGSI v1 API            | Not yet defined                                       | Not yet defined               |
+| Support to Node.js v4  | 2.8.1                                                 | December 2018                 |
+| Support to Node.js v6  | 2.9.0                                                 | May 2019                      |
+| Support to Node.js v8  | 2.12.0                                                | April 2020                    |
```
Comment (fgalan) [inline doc/deprecated.md:44]:
```suggestion
| Support to Node.js v4  | 2.8.1                                                 | December 19th, 2018                 |
| Support to Node.js v6  | 2.9.0                                                 | May 22nd, 2019                      |
| Support to Node.js v8  | 2.12.0                                                | April 7th, 2020                    |
```

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**